app/bundles/EmailBundle/Config/config.php

```diff
@@ -90,6 +90,9 @@
             'mautic.emailbuilder.subscriber'         => array(
                 'class' => 'Mautic\EmailBundle\EventListener\BuilderSubscriber'
             ),
+            'mautic.emailtoken_subscriber'     => array(
```

Comment:
:-) A true OCD comment.  Yes, indeed and it would have bothered me too if I had noticed it. Fixed.